oslabs-beta/Kubiquity
# Add Loading component

WHAT DOES THIS PR DO? 
1. Adds a loading animation. 
2. Adds a Loading component. 
3. Imports that component into the Nodes and Errors. 

HOW TO TEST THIS PR:
1. Run `gh pr checkout 21 && npm install && npm run dev`
2. When loading the app, verify that the table has the loading component before errors are fetched. 
3. Verify the loading icon generally looks good. 

ADDITIONAL NOTES: 
1. Because of the PR backlog, this will likely have merge conflicts for me to resolve before merging in.
